controllers/volumereplicationgroup_controller.go

```diff
@@ -589,9 +627,12 @@ func (v *VRGInstance) preparePVCForVRProtection(pvc *corev1.PersistentVolumeClai
 
 	// if the PVC is not bound yet, don't proceed. If PVC is deleted, it will eventually go away
 	if pvc.Status.Phase != corev1.ClaimBound {
-		log.Info("Requeuing as PersistentVolumeClaim is not bound", "pvcPhase", pvc.Status.Phase)
+		log.Info("Skipping handling of VR as PersistentVolumeClaim is not bound", "pvcPhase", pvc.Status.Phase)
```

Comment:
The return from this function will delete the VR in function `reconcileVRsForDeletion` as skip is true and requeue is false.

As the delete of the VR on skip is not really effective, I suggest we remove that in `reconcileVRsForDeletion` and deal with it as part of issue #52

Reply:
ACK